application/frontend/src/pages/releases/detail/access-box/object-signing-form.tsx

```diff
@@ -0,0 +1,64 @@
+import { CSRFInputToken } from "../../../../components/csrf-token";
+import { ReleaseTypeLocal } from "../../shared-types";
+import { ObjectStoreRecordKey } from "@umccr/elsa-types/schemas";
+
+type Props = {
+  releaseKey: string;
+  releaseData: ReleaseTypeLocal;
+};
+
+type TsvColumnCheckProps = {
+  field: string;
+};
+
+const TsvColumnCheck: React.FC<TsvColumnCheckProps> = ({ field }) => (
+  <div key={field} className="flex items-center gap-2">
```

Comment:
You copied my code from `manifest-form.tsx`, and my code is bad :sweat_smile: 

```suggestion
  <div className="flex items-center gap-2">
```